src/lib/linting.js

```diff
@@ -6,24 +6,24 @@
  * It receives the parsed AST and rules from ../svglint.js, and then runs each
  *   rule and gathers the results.
  */
-import { EventEmitter } from "events";
-import path from "path";
-import process from "process";
-import * as cheerio from "cheerio";
-import * as parse from "./parse.js";
-import Reporter from "./reporter.js";
-import Logger from "./logger.js";
+import {EventEmitter} from 'node:events';
+import path from 'node:path';
+import process from 'node:process';
+import * as cheerio from 'cheerio';
```

Comment:
`cheerio` provides a `exports.default` and a `exports =` from `1.0.0-rc.6`, which is the lowest version used by SVGLint. I've tested the next change with `1.0.0-rc.6` and works fine:

```suggestion
import cheerio from 'cheerio';
```